README.md

```diff
@@ -43,15 +43,16 @@ and additional sections `[server]`, `[plot]` and `[dataframe]`, or to include ev
 pip install intake[complete]
 ```
 
-Note that you may well need specific drivers and other plugins, which usually have additional 
-dependencies of their own. 
+Note that you may well need specific drivers and other plugins, which usually have additional
+dependencies of their own.
 
 Development
 -----------
- * Create development Python environment, ideally with `conda`. The requirements can be found in the
-   recipe in the `conda/` directory of this repo or in the sister 
-   [feedstock](https://github.com/conda-forge/intake-feedstock)
- * Install using `pip install -e .[complete]`
- * Add `pytest` to the environment to be able to run tests
+ * Create development Python environment with the required dependencies, ideally with `conda`.
+   The requirements can be found in the yml files in the `scripts/ci/` directory of this repo or in the sister
+   [feedstock](https://github.com/conda-forge/intake-feedstock).
```

Comment:
Actually, I would remove the reference to the feedstock. It has a slightly different set of packages and would need the user to do an additional fetch.